rfc6962-bis.xml

```diff
@@ -656,7 +655,7 @@ updated on the fly.
 (see <xref target="cert_sctlist_extension"/>). This mechanism allows the use of
 unmodified TLS servers, but, because the included SCTs cannot be changed without
 re-issuing the certificate, increases the risk that the certificate will be
-refused if any of the SCTs become invalid.
+considered non-compliant if some of the SCTs become invalid (by the client not recognizing the logs they were issued from anymore).
```

Comment:
I think so; I've added "with this document" due to complaints in another trac ticket, but if we're all in agreement it's superfluous then we can leave it this way and re-open the discussion if anybody brings up the issue of definition of compliance with the new wording.